ext/java/nokogiri/XmlXpathContext.java

```diff
@@ -114,9 +118,15 @@ public IRubyObject evaluate(ThreadContext context, IRubyObject expr, IRubyObject
         if (!handler.isNil()) {
```

Comment:
This condition should now always be true (i.e. handler will always be an instance of `XPathFunctions`).  Should I remove this `if`?